packages/skeleton-react/src/lib/components/Segment/types.ts

```diff
@@ -1,6 +1,21 @@
 // Segment Control Types
 
+// Context ---
+
+export interface SegmentContextState {
+	value: string;
+	name: string;
+	onSelectionHandler: (value: string) => void;
+}
+
+// Components ---
+
 export interface SegmentProps extends React.PropsWithChildren {
+	/** Set the group value, which determines selection state. */
+	value?: string;
+	/** Provide the shared input name. */
+	name: string;
```

Comment:
I'd assume because of this: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#name

Reply:
True but MDN is assuming you aren't using a framework where people might only want client side forms where values are directly bound/grabbed from the DOM instead of submitted via form. It wouldn't make sense to add a `name` in that case.

Reply:
I figured it's better to reinforce best practices rather than troubleshoot why form submission isn't working for folks because they left off the name.

Reply:
Do you mind if I update the Listbox PR with this aswell?

Reply:
Go for it